src/main/java/hudson/plugins/jira/Updater.java

```diff
@@ -348,10 +406,29 @@ static void findIssues(AbstractBuild<?, ?> build, Set<String> ids, Pattern patte
         }
     }
 
-    private static final Logger LOGGER = Logger.getLogger(Updater.class.getName());
-
-    /**
-     * Debug flag.
-     */
-    public static boolean debug = false;
+    private List<ChangeLogSet<? extends Entry>> getChanges(Run<?,?> run)
+    {
+        if(run instanceof AbstractBuild)
+        	return ((AbstractBuild<?,?>) run).getChangeSets();
+        else if(run instanceof WorkflowRun)
+        	return ((WorkflowRun)run).getChangeSets();
```

Comment:
Pending this aspect of [JENKINS-24141](https://issues.jenkins-ci.org/browse/JENKINS-24141), I recommend looking for a `getChangeSets` method via reflection, rather than adding (even an `optional`) plugin dependency on `workflow-job`.